drucker_dashboard/apis/api_kubernetes.py

```diff
@@ -929,7 +933,7 @@ def put(self):
     def post(self):
         """add_kubernetes"""
         args = kube_file_parser.parse_args()
-        config_path = "{0}/{1}.config".format(api.dashboard_config.DIR_KUBE_CONFIG, uuid.uuid4().hex)
+        kube_config_filename = "{0}.config".format(uuid.uuid4().hex)
```

Comment:
The new naming is a bit confusing because `Kubernetes` has original argument named `config_path`.

I suggest to keep `config_path` as-is where it is used for `Kubernetes`. On the other hand, for DB in upper lines name it like: 
`full_config_path = get_full_config_path(kobj.config_path)`

Reply:
I will use `full_config_path` for this. Thank you!